includes/class-wc-stripe-order-helper.php

```diff
@@ -480,6 +667,7 @@ public function add_payment_intent_to_order( string $payment_intent_id, WC_Order
 	 * @return void
 	 */
 	public function set_payment_awaiting_action( WC_Order $order, bool $save = true ): void {
+		$this->update_order_meta( $order, self::META_STRIPE_PAYMENT_AWAITING_ACTION, true );
 		$order->update_meta_data( self::META_STRIPE_PAYMENT_AWAITING_ACTION, wc_bool_to_string( true ) );
```

Comment:
Should we make only one update here? It looks like we're performing two redundant upates.
```suggestion
		$this->update_order_meta( $order, self::META_STRIPE_PAYMENT_AWAITING_ACTION, wc_bool_to_string( true ) );
```